torch/csrc/lazy/core/tensor_impl.cpp

```diff
@@ -82,6 +87,18 @@ LTCTensorImpl::LTCTensorImpl(LazyTensor&& tensor)
   // This is a temporary fix for a PyTorch core issue,
   // according to https://github.com/pytorch/xla/pull/2682.
   is_non_overlapping_and_dense_ = false;
+
+  auto rank = tensor_->shape().Get().sizes().size();
+  std::vector<int64_t> sym_ints;
```

Comment:
allocating the vector here is directly, just allocate the correct rank and then write in the sizes one-by-one directly into `sizes_and_strides_.size_at_unchecked()`